package.json

```diff
@@ -45,7 +45,6 @@
     "rollup-plugin-eslint": "^7.0.0"
```

Comment:
It seems that the `rollup` version higher than the v4 [needs](https://rollupjs.org/migration/#prerequisites) at least NodeJS v18. We support the NodeJS version >=16.14.0 (see the engines section in the `package.json`), so I think we should downgrade the `rollup`.

Reply:
I will downgrade this version of `rollup` here: https://github.com/highcharts/node-export-server/pull/426